client/src/components/atoms/EditTestimonialCard.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FunctionComponent, useState } from "react";
+import { Button } from "../atoms/Button";
+import DefaultImage from "../../assets/grey-background.png";
+import OverlayImage from "../../assets/image-upload.png";
+import TextArea from "../atoms/TextArea";
+import UploadImageModal from "../organisms/UploadImageModal";
+
+interface Props {
+    id: number;
+    imagePath: string;
+    testimonial: string;
+    showDelete: boolean;
+    onDelete: (id: number) => void;
+    onSave: (id: number, imagePath: string, testimonial: string) => void;
+    onCancel: (id: number) => void;
+}
+
+const EditTestimonialCard: FunctionComponent<Props> = (props: Props) => {
+    const [testimonial, setTestimonial] = useState(props.testimonial);
+    const [imagePath, setImagePath] = props.imagePath === "" ? useState(DefaultImage) : useState(props.imagePath);
+    const [showImagePicker, setShowImagePicker] = useState(false);
+    const minNumChars = 80;
+    const [testimonialError, setTestimonialError] = useState(false);
+    const [imageError, setImageError] = useState(false);
+
+    const onChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+        setTestimonial(event.target.value);
+        if (testimonial.length > minNumChars) {
+            setTestimonialError(false);
+        }
+    };
+    const textAreaProps = {
+        isErroneous: false,
+        onChange: onChange,
+        value: testimonial,
+        placeholder: "",
+        label: "Edit Client Story",
+        maxNumChars: 400,
+        minNumChars: minNumChars
+    };
+
+    const primaryBtnProps = {
+        className: "update-button",
+        text: "Update",
+        onClick: () => {
+            let error = false;
+            if (imagePath === DefaultImage) {
+                setImageError(true);
+                error = true;
+            }
+            if (testimonial.length < minNumChars) {
+                setTestimonialError(true);
+                error = true;
+            }
+            if (error) return;
+            setImageError(false);
+            setTestimonialError(false);
+            props.onSave(props.id, imagePath, testimonial);
+        },
+        copyText: ""
+    };
```

Comment:
personally i think it's cleaner to place the props directly in the jsx because it makes it easier to identify the purpose of the code

to prevent the object from being bulky you can define a separate `const onUpdate = ...` with the onClick logic, then set `onClick: onUpdate` in the props